benches/aarch64/mod.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Just realized, we also need to exclude the `benches` directory when computing the coverage. Can you also update the coverage_config file?

Reply:
Done, the coverage test actually fails with the newer `rustc`, so I had to decrease the expected value :(